<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="pastFilesWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QListWidget" name="fileList">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="snapShotButton">
           <property name="text">
            <string>Snapshot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_1d">
          <attribute name="title">
           <string>1D Plots</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="plot_1d" native="true"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="status_1d">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Normalizer</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="norm_cb"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2d">
          <attribute name="title">
           <string>2D Plots</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="rightMargin">
              <number>36</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="plot_2d" native="true"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="status_2d">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="showDifferenceButton">
                   <property name="text">
                    <string>Difference</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="showSumButton">
                   <property name="text">
                    <string>Sum</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="updateReferenceButton">
           <property name="text">
            <string>Update Reference</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetReferenceButton">
           <property name="text">
            <string>Reset Reference</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/print</normaloff>:/print</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="toolTip">
    <string>Print current plot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionUnzoom">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/viewfull</normaloff>:/viewfull</iconset>
   </property>
   <property name="text">
    <string>&amp;Unzoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionSavePlot">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as Image...</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save as Data...</string>
   </property>
  </action>
  <action name="actionLogScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Logarithmic Y scale</string>
   </property>
   <property name="iconText">
    <string>Log Scale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionColormap">
   <property name="text">
    <string>&amp;Colormap</string>
   </property>
   <property name="toolTip">
    <string>Select colormap</string>
   </property>
  </action>
  <action name="actionROI">
   <property name="text">
    <string>&amp;Region Of Interest</string>
   </property>
   <property name="toolTip">
    <string>Open region of interest in seperate window</string>
   </property>
  </action>
  <action name="actionMarkCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Center Mark</string>
   </property>
   <property name="toolTip">
    <string>Mark center of detector image</string>
   </property>
  </action>
  <action name="actionKeepRatio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep ratio</string>
   </property>
   <property name="toolTip">
    <string>Keep aspect ratio of selection rectangle</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../../../resources/nicos-gui.qrc">
     <normaloff>:/open</normaloff>:/open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
   <property name="toolTip">
    <string>Activate Lines</string>
   </property>
  </action>
  <action name="actionSymbols">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show symbols</string>
   </property>
   <property name="toolTip">
    <string>Show symbols</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>&amp;Load data</string>
   </property>
  </action>
  <action name="actionSubtractBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Subtract Background</string>
   </property>
   <property name="toolTip">
    <string>Subtract background</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections/>
</ui>
